helm/ph-ee-engine/templates/connector-mpesa.yaml

```diff
@@ -125,7 +125,7 @@ spec:
             - name: "ACCOUNTS_ROSTER_CURRENCY"
               value: "{{ .Values.mpesa.paybill.roster.currency }}"
             - name: "ACCOUNT_HOLDING_INSTITUTION_ID"
-              value: "{{ .Values.mpesa.paybill.accountHoldingInstitutionId }}"
+              value: "{{ .Values.mpesa.paybill_accountHoldingInstitutionId }}"
```

Comment:
if you are giving this as env variable  dot (.) is fine.  keep it as  .Values.mpesa.paybill.accountHoldingInstitutionId

Reply:
- name: "paybill_accountHoldingInstitutionId"
   value: "default"
            add these in values.yml as a workaround